newsroom/web/default_settings.py

```diff
@@ -144,6 +144,7 @@
     "newsroom.email_templates",
     "newsroom.search",
     "newsroom.notifications.commands",
+    "newsroom.mgmt_api",
```

Comment:
This shouldn't be needed. CORE_APPS is the old way, and this new module does not provide any `init_app`. So I believe this can be safely removed from this line